src/app/questionnaire-item/questionnaire-form/questionnaire-form.component.html

```diff
@@ -1,4 +1,4 @@
-<form (ngSubmit)="submitQuestionnaire.next()">
+<form (ngSubmit)="saveAsDraft.next()">
```

Comment:
Shouldn't we use the type submit button to trigger the submit action?

Reply:
IMO we could also solely rely on type button buttons.

Reply:
I do not want to accidentally trigger a submit event and therefore I changed the default action of the submit. But yes, I think no submit at all would be an option as well.